Gruntfile.js

```diff
@@ -37,10 +37,21 @@ module.exports = function(grunt) {
         }
       }
     },
+    postcss: {
+      options: {
+        map: true,
```

Comment:
@josephgknox: I have now set this to true in order to preserve the sass maps.

Reply:
@sherakama awesome! I had originally set it to false as the sass task was writing the maps. Based on order of operations (and now that these things happened as part of a single command) this should've been set to true. Thanks!